src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+import duke.tasks.Task;
+
+/**
+ * Represents the storage component of the Duke chatbot. The Storage class handles then reading and writing
+ * of the file that stores the content of the Duke chatbot. Each <code>Storage</code> object contains a
+ * <code>Path</code> and a <code>File</code>
+ */
+public class Storage {
+    private Path path;
+    private File f;
```

Comment:
```suggestion
    private File file;
```
Try to use more descriptive variable names for variables with large scopes (e.g. instance vars)